.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest]
+        plan:
+        - { build: stack, resolver: "--resolver lts-14.25"  }
+        include: []
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install OS Packages
+      uses: mstksg/get-package@v1
+      with:
+        apt-get: libgmp-dev libssl-dev libtinfo-dev libsystemd-dev zlib1g-dev
```

Comment:
small note: I need libssl-dev and libsystemd-dev (and probably most of the other except libgmp-dev) for building the consensus and network libraries in cardano-transactions.
I think that here, only `libgmp-dev` and `libssl-dev` are necessary for building cryptonite & cardano-crypto.

Reply:
yes, `libgmp-dev` and `libssl-dev` are enough